quantum/split_common/transport.c

```diff
@@ -91,6 +92,9 @@ bool transport_master(matrix_row_t matrix[]) {
         }
     }
 #    endif
+
+    // If this is the master, a USB connection was established previously
+    i2c_buffer->usb_connected = true;
```

Comment:
This buffer is not automatically synced so will never be transferred thus always false on i2c. It would need extra logic like the WPM sync.

Reply:
I see, I will have a look.

Reply:
I added this extra logic in https://github.com/qmk/qmk_firmware/pull/9742/commits/f704c02cb3617dc1821824a5879ee173199cf123